Fix Overviews (#6894)
DevExpress/devextreme-documentation: concepts/05 UI Components/HtmlEditor/02 Overview/10 Key Features.md

- **Mentions**     
[Insert mentions](https://js.devexpress.com/Demos/WidgetsGallery/Demo/HtmlEditor/Mentions/) to reference others within text conversations.

- **Tables**    
Users can [insert and modify tables](/concepts/05%20UI%20Components/HtmlEditor/00%20Getting%20Started%20with%20HtmlEditor/30%20Work%20with%20Tables.md '/Documentation/Guide/UI_Components/HtmlEditor/Getting_Started_with_HtmlEditor/#Work_with_Tables') with built-in UI elements. You can allow users to [resize tables](/Documentation/ApiReference/UI_Components/dxHtmlEditor/Configuration/tableResizing/).

- **Built-in Format Customization**     
You can [modify the existing formats](/concepts/05%20UI%20Components/HtmlEditor/10%20Formats/33%20Customize%20Built-In%20Formats%20and%20Modules '/Documentation/Guide/UI_Components/HtmlEditor/Formats/#Customize_Built-In_Formats_and_Modules') to fit specific requirements.

- **Advanced Format Conversion**    

> An HTML Editor default value type is HTML. You can [convert](/Documentation/ApiReference/UI_Components/dxHtmlEditor/Configuration/converter/) the value from HTML to another markup language, for instance, to Markdown.

Reviewer: ```suggestion
The HTML Editor's default value type is HTML. You can [convert](/Documentation/ApiReference/UI_Components/dxHtmlEditor/Configuration/converter/) the value from HTML to another markup language, for instance, Markdown.
```

    #include btn-open-demo with {
        href: "https://js.devexpress.com/Demos/WidgetsGallery/Demo/HtmlEditor/MarkdownSupport/"
    }

- **Mail Merge**     
Integrate [variables](/api-reference/10%20UI%20Components/dxHtmlEditor/1%20Configuration/variables '/Documentation/ApiReference/UI_Components/dxHtmlEditor/Configuration/variables/') to be replaced with real data during text processing.

- **Adaptive Toolbar**     
The HTML Editor's built-in [adaptive toolbar](/concepts/05%20UI%20Components/HtmlEditor/00%20Getting%20Started%20with%20HtmlEditor/20%20Configure%20the%20Toolbar.md '/Documentation/Guide/UI_Components/HtmlEditor/Getting_Started_with_HtmlEditor/#Configure_the_Toolbar') contains buttons and drop-down menus that allow users to edit and format content. You can use [predefined items](/concepts/05%20UI%20Components/HtmlEditor/20%20Toolbar/00%20Predefined%20Items '/Documentation/Guide/UI_Components/HtmlEditor/Toolbar/Predefined_Items/') or create [custom ones](/concepts/05%20UI%20Components/HtmlEditor/20%20Toolbar/20%20Add%20a%20Custom%20Item.md '/Documentation/Guide/UI_Components/HtmlEditor/Toolbar/Add_a_Custom_Item/'). 

- **Data Validation and Spellcheck**    

> HTML Editor supports browser spellcheck functionality. The component includes various [validation rules](https://js.devexpress.com/Demos/WidgetsGallery/Demo/Validation/Overview/) and real-time [validation feedback](/Documentation/ApiReference/UI_Components/dxHtmlEditor/Configuration/#validationStatus). Adjust [error message position](/Documentation/ApiReference/UI_Components/dxHtmlEditor/Configuration/#validationMessagePosition) as needed.

Reviewer: ```suggestion
HTML Editor supports browser spellcheck functionality. The component includes various [validation rules](https://js.devexpress.com/Demos/WidgetsGallery/Demo/Validation/Overview/) and real-time [validation feedback](/Documentation/ApiReference/UI_Components/dxHtmlEditor/Configuration/#validationStatus). You can also adjust the [error message position](/Documentation/ApiReference/UI_Components/dxHtmlEditor/Configuration/#validationMessagePosition).
```
